triplet/main.py

```diff
@@ -151,7 +145,13 @@ def convert_and_transform(img, transform):
         return img_a, img_p, img_n
 
     def __len__(self):
-        return self.triplets.shape[0]
+        try:
+            if self.train:
+                return self.triplets.shape[0]
+            else:
+                return self.matches.shape[0]
+        except:
+            pass
 
```

Comment:
Can you share a full stack trace? I need to see where this method was called from.